src/components/molecules/NavSearchBar/NavSearchBar.tsx

```diff
@@ -79,72 +82,78 @@ const NavSearchBar = () => {
     <div className="nav-search-links">
       <div className="nav-search-icon" />
       <NavSearchBarInput value={searchQuery} onChange={setSearchQuery} />
+
       {isTruthy(searchQuery) && (
         <div className="nav-search-close-icon" onClick={clearSearchQuery} />
       )}
-      {isTruthy(numberOfSearchResults) && (
-        <>
-          <div className="nav-search-results">
-            <div className="nav-search-result-number">
-              <b>{numberOfSearchResults}</b> search results
-            </div>
-            {searchResult.rooms.map((room, index) => {
-              return (
+
+      {numberOfSearchResults > 0 && (
+        <div className="nav-search-results">
+          <div className="nav-search-result-number">
+            <b>{numberOfSearchResults}</b> search results
+          </div>
+
+          {/* @debt we really shouldn't be using the index as part of the key here, it's unstable.. but rooms don't have a unique identifier */}
+          {searchResult.rooms.map((room, index) => {
+            return (
+              <div
+                className="row"
+                key={`room-${room.title}-${index}`}
```

Comment:
While we shouldn't be using index at all, this is sort of a hacky workaround since rooms don't technically have a unique identifier.

It's far less likely that we will find a different room that has the exact same `title` that ends up in the same `index`, so while it's still possible for this to be incorrect.. it's WAY less likely to occur.